### ISSUE TITLE
Plugins (NSM)

### ISSUE THREAD

**Issue by grammoboy2** [Houston4444/RaySession#207]
I saw you've plans to provide a way to include plugins into a session.

Wouldn't be a good first step to add NSM support to Jalv, Clap-Host (by @free-audio) and such? Adding a plugin to Raysession could then launch Jalv / Clap-Host, ... (in the background).

**Comment by Houston4444** [comment]
Exactly, this or carla-single. First, we need a such software to be NSM compatible with optional-gui capability, then we could give an user quick access to it.

**Comment by grammoboy2** [comment]
A while ago, I was exploring if a application like jalv would work under NSM, now that is has a plugin selector as GUI. It also helped me to test restoring JACK metadata via a JACKPatch like application. I don't see myself working on this any time soon, so I'll publish it. 
Warning: this was just prototyping by a non-C coder, but it could help to identify the places in the code where NSM stuff could/should happen. 
https://github.com/grammoboy2/jalv-nsm
